### PR TITLE
Require Maven 3.5.4+

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
README to fix

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
     with:
       ff-goal: '-P run-its install'         # site use project version for reporting
       verify-goal: '-P run-its install'     # should be the same as for first build
+      maven-matrix: '[ "3.5.4", "3.9.1" ]'  # Maven versions matrix for verify builds
```
Comment (slawekjaranowski) [inline .github/workflows/maven.yml:29]: build will take about 1 hour 😄

Comment (slachiewicz) [reply .github/workflows/maven.yml:29]: yes, but at last I would like to test somehow once

**Comment by ajarmoniuk** [comment]
Currently, version also needs to be updated here: https://github.com/mojohaus/versions/blob/6cd759ce2b1e884ebbbf3dece70b156d1b102753/versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayPluginUpdatesMojo.java#L128

However, I think that this assumption is bogus: minimum maven version of the POM doesn't have to be the same as the minimum version required to run versions-maven-plugin.

**Pull request closed without merging**